scss/modules/_grid.scss

```diff
@@ -9,38 +9,73 @@ $grid-col-padding: 15px;
   .container-fluid { width: 100%; }
 
   .container {
+    margin: 0 auto;
     max-width: $site-max-width;
-    margin-right: auto;
-    margin-left: auto;
   }
 
   .row {
-    @include clearfix-overflow-visible;
-    margin-right: -$grid-col-padding;
-    margin-left: -$grid-col-padding;
+    margin-right: auto;
+    margin-left: auto;
+    
+    &:before,
+    &:after,
+    & &:before,
+    & &:after  {
+      content: " ";
+      display: table;
+    }
+
+    &:after,
+    & &:after {
+      clear: both;
+    }
+
+    & & {
+      margin-right: -$grid-col-padding;
+      margin-left: -$grid-col-padding;
+    }
+
   }
 
   .col {
     padding-right: $grid-col-padding;
     padding-left: $grid-col-padding;
   }
 
-  .col-opposite { float: right; }
-
   @for $i from 1 through $grid-cols {
     .col-#{$i},
     .col-sm-#{$i} {
       float: left;
       width: (100% / $grid-cols) * $i;
       padding-right: $grid-col-padding;
       padding-left: $grid-col-padding;
+      position: relative;
```

Comment:
Maybe better to target .col with [class^="col-"] {} ? Sometimes .col is omitted when declaring columns to elements in projects. I'd like to keep it as it is. Eg, with position relative at two places in the code.